app/src/test/java/org/mozilla/fenix/tabstray/TabsTrayStoreReducerTest.kt

```diff
@@ -69,4 +70,18 @@ class TabsTrayStoreReducerTest {
 
         assertEquals(expectedState, resultState)
     }
+
+    @Test
+    fun `WHEN UpdateSyncedTabs THEN synced tabs are added`() {
```

Comment:
@jonalmeida Hm.. it's actually interesting to see that we have a Reducer and Store test separately. Usually I see we use the Store test to test the underlying Reducer. Is there any benefit from having them separate like this? I can imagine we end up picking and choosing which actions are dispatched and resulting state in different files.

Reply:
I missed this comment.

What does the store add that needs to be tested? In most of the stores, the store doesn't do anything for us within the structure. The logic is within the state/reducer. Put state in, reduce, test state out.

The store holds an executor and provides single access to the state and to dispatch actions. Nothing worth testing in tabs tray.